flocker/node/agents/ebs.py

```diff
@@ -58,11 +100,95 @@ def ec2_client(region, zone, access_key_id, secret_access_key):
     config.set('Boto', 'num_retries', BOTO_NUM_RETRIES)
     config.set('Boto', 'metadata_service_num_attempts', BOTO_NUM_RETRIES)
 
+    # Set Boto debug level to BOTO_DEBUG_LEVEL:
+    # "1": basic debug messages
+    config.set('Boto', 'debug', BOTO_DEBUG_LEVEL)
```

Comment:
Agree on the impact on other modules in the process. Filed https://clusterhq.atlassian.net/browse/FLOC-1962 and linked to it from ebs.py.
Debug level "1" might be useful in troubleshooting operations from other modules as well, but as you pointed out, would be good to quantify impact. Thanks for noticing!